src/AddSupplier.tsx

```diff
@@ -0,0 +1,24 @@
+import { VStack, StackDivider, Box, Flex } from '@chakra-ui/react';
+import AddSupplierTittle from './Supplier/AddSupplierTittle';
+import AddSupplierButtons from './Supplier/AddSupplierButtons';
+import AddSupplierData from './Supplier/AddSupplierData/AddSupplierData';
+
+function Form() {
+  return (
+    <Flex w="100%" p="4">
+      <Box w="26%" h="520" bg="gray.200" />
+      <Box w="2%" h="10" bg="white" />
```

Comment:
Couple lines below you define `h` property as `h="400px"` so with `px` suffix.
Use one convention.

Reply:
You can also consider using `em` or `rem` instead of pixels.
It will better adjust to user browser (f.e when scaling).